.github/workflows/test_run.yml

```diff
@@ -0,0 +1,34 @@
+name: "Test run"
+
+on: [ 'pull_request' ]
+
+jobs:
+  test_run:
+    runs-on: ubuntu-latest
+    
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"
+
+      - name: Checking dir
+        run: echo $(ls)
+
+      - name: Update apt
+        run: sudo apt update
+
+      - name: Install pip
+        run: sudo apt install pip
+        
+      - name: install dependencies
+        run: sudo pip install requests simplejson orionutils pyyaml
+
+      - name: install dependencies_olny_simplejson
+        run: sudo pip install simplejson
```

Comment:
we should not be installing anything other than galaxykit

if galaxykit dependencies are declared correctly, pip will install them with galaxykit